README.md

```diff
@@ -1,2 +1,97 @@
+[![Docker Repository on Quay](https://quay.io/repository/bcdev/xcube/status "Docker Repository on Quay")](https://quay.io/repository/bcdev/xcube)
+[![Build Status](https://travis-ci.com/dcs4cop/xcube-conda.svg?branch=master)](https://travis-ci.com/dcs4cop/xcube-conda)
+[![Anaconda-Server Badge](https://anaconda.org/bc-dev/xcube/badges/latest_release_date.svg)](https://anaconda.org/bc-dev/xcube)
+[![Anaconda-Server Badge](https://anaconda.org/bc-dev/xcube/badges/platforms.svg)](https://anaconda.org/bc-dev/xcube)
+
 # xcube-conda
 Repository for deploying xcube as a conda package
+
+## Description
+This repository combines a conda and docker build as well as their deployment.
+
+REMARK: The current setup would allow integrating the deployment to the repo xcube.
+
+It follows these steps:
+
+- Change versions in meta.yml, Dockerfile, .travis.yml
+- push to a branch and await build
+- Once build is successful create PR
+- Once PR is accepted the master will be build on travis
+- If that is successful, create a tag or release (and push it if you use the command line)
+- The tag will trigger a travis build. At the end of the travis build, two
+  deployment steps follow (only on tag)
+  - upload all conda packages to the anaconda cloud
+  - trigger a build on quay.io 
+- Once the deployment steps have finished and the docker image has been built on quay
+  you can:
+  - use ```conda install -c bc-dev xcube``` to install xcube in a conda environment
+  - user docker image ```quay.io/dzelge/xcube:[version]``` e.g.  ```docker pull quay.io/dzelge/xcube:[version]```
+
+TODO: Change the quay repo and travis accounts and setup the deploy key accordingly  
+
+    
+## Usage for conda images
+
+```bash
+
+conda install -c bc-dev xcube[=version]
+# OR
+conda create pn xcube -c bc-dev xcube[=version]
+
+```
+
+Or use specify the xcube package in an ```environment.yml```.
+
+
+## Usage for Docker images
+
+TODO: Describe how to not be a root user in the docker container
```

Comment:
Why is this important here? Describe please.
If it really is important, then remove TODO and do provide description.